src/components/DatePicker.vue

```diff
@@ -504,9 +511,27 @@
           return D + ['th', 'st', 'nd', 'rd'][D % 10 > 3 ? 0 : (D - D % 10 !== 10) * D % 10];
         }
       };
+      let checkinDate = this.formatDate(new Date(this.checkIn) , 'YYYYMM');
+      let startDate = this.formatDate(new Date(this.startDate), 'YYYYMM');
```

Comment:
the same here, default to `const` and use `let` only when it will be reassigned

Reply:
it's removed